spec/models/work_package/work_package_acts_as_journalized_spec.rb

```diff
@@ -379,9 +379,8 @@
 
         it 'does not set an upper bound to the already existing journal' do
           work_package.save
-
-          expect(work_package.last_journal.validity_period.last)
-            .to eql(Float::INFINITY)
+          expect(work_package.last_journal.validity_period.end)
+            .to be_nil
```

Comment:
Your grasp on Journals is way further along than mine. All I'm wondering is if this has any behavioral implication

Reply:
We have a good coverage for testing the aggregation periods so I'm confident that it doesn't change the behaviour. Additionally I verified the database entries and they are also correct.